src/migration-wizard/steps/connect/sources-table/empty-state/DiscoverySourceSetupModal.tsx

```
@@ -25,17 +24,19 @@ import {
-  onStartDownload: () => void;
-  onAfterDownload: () => Promise<void>;
-};
+export namespace DiscoverySourceSetupModal {
+  export type Props = {
+    isOpen?: boolean;
+    isDisabled?: boolean;
+    onClose?: (event?: KeyboardEvent | React.MouseEvent) => void;
+    onStartDownload: () => void;
+    onAfterDownload: () => Promise<void>;
+  };
+}
```

ESLint

[error] ES2015 module syntax is preferred over namespaces.

(@typescript-eslint/no-namespace)